osgtest/tests/test_55_condorce.py

```diff
@@ -150,6 +150,7 @@ def test_07_ping_with_gums(self):
 gums.authz=https://%s:8443/gums/services/GUMSXACMLAuthorizationServicePort
 ''' % (hostname, hostname)
 
+        core.config['condor-ce.lcmapsdb'] = os.path.join('/', 'etc', 'lcmaps.db')
         core.config['condor-ce.gums-properties'] = '/etc/gums/gums-client.properties'
```

Comment:
Minor: in the same block here, paths are just written out, eg, `/etc/lcmaps.db` instead of `os.path.join('/', 'etc', 'lcmaps.db')`